package.json

```diff
@@ -95,7 +95,7 @@
   "dependencies": {
     "@cocos/box2d": "1.0.1",
     "@cocos/cannon": "1.2.8",
-    "@cocos/ccbuild": "2.0.3-alpha.7",
+    "@cocos/ccbuild": "2.0.3-alpha.18",
```

Comment:
From alpha.7 to alpha.18? Is it a mistake? From alpha.7 to alpha.8?

Reply:
It's not a mistake. It's a long time we don't upgrade the `ccbuild` module.
You could refer to https://github.com/cocos/cocos-ccbuild/blob/master/package.json#L3